falcon/asgi/request.py

```diff
@@ -450,6 +452,7 @@ def __init__(self, scope, receive, options=None):
 
         self._stream = None
         self._receive = receive
+        self._first_event = first_event
```

Comment:
Good catch. There should be a local slot for it for the sake of efficiency, even though this accidentally works because the parent WSGI Request class defines `__dict__` in its own `__slots__`.

Reply:
I was convinced that only response did have it. I guess the mystery is resolved then